indy_node/server/domain_req_handler.py

```diff
@@ -421,18 +421,18 @@ def add_query_handler(self, txn_type, handler: Callable):
 
     def handleGetNymReq(self, request: Request):
         nym = request.operation[TARGET_NYM]
-        nymData = self.idrCache.getNym(nym, isCommitted=True)
```

Comment:
Why don't we use IDR Cache anymore?

Reply:
Because same data is present in state trie too.